lib/VeresOneClient.js

```diff
@@ -128,17 +137,26 @@ class VeresOneClient {
     let result;
     try {
       result = await httpClient.post(acceleratorUrl, {
-        json: operation, headers, httpsAgent: this.httpsAgent
+        json: {operation}, headers, agent: this.httpsAgent
```

Comment:
@dmitrizagidulin this is the only major change in functionality. the post to the accelerator now is `{operation}` where previously it was just `operation`. That does need to be reviewed.

Reply:
Noted, k.